client/my-sites/patterns/components/grid-gallery/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { Button } from '@wordpress/components';
+import { PatternsSection } from 'calypso/my-sites/patterns/components/section';
+import type { CSSProperties } from 'react';
+
+import './style.scss';
+
+type PatternsGridGalleryProps = {
+	title: string;
+	description: string;
+	itemMinWidth?: number;
+	list: {
+		name: string;
+		number: number;
+		image: string;
+		link: string;
+	}[];
+};
+
+export const PatternsGridGallery = ( {
+	title,
+	description,
+	itemMinWidth = 287,
+	list,
+}: PatternsGridGalleryProps ) => {
+	return (
+		<PatternsSection title={ title } description={ description }>
+			<div
+				className="patterns-grid-gallery"
+				style={ { '--item-min-width': `${ itemMinWidth }px` } as CSSProperties }
+			>
+				{ list.map( ( { name, number, image, link } ) => (
+					<Button className="patterns-grid-gallery--item" href={ link }>
```

Comment:
```suggestion
					<a className="patterns-grid-gallery--item" href={ link }>
```
There's no requirement to use `Button` for client-side navigation purposes. We could probably use a native `a` here instead.